ecosystem/sep-0028.md

```diff
@@ -0,0 +1,37 @@
+## Preamble
+
+```
+SEP: 28
+Title: Ease burden on new users
+Author: @buhrmi
+Track: Informal
+Status: Draft
+Created: 2019-11-26
+Discussion: (see mailing list, waiting for approval)
```

Comment:
@buhrmi Could you update this to point to the mailing list conversation? I'm not able to identify the conversation this is linked to.